### PR TITLE
SHT3x & CCS811 (m5 stack TFT also added)

### PR DESCRIPTION
**Pull request by stripeyhorse**
added support for SHT3x and CCS811

### CODE REVIEW

**Comment by 1technophile** [comment]
OpenMQTTGateway.ino and user_config.h are detected as completely changed by github, maybe there was a change in file encoding or something else. Would it be possible to recover the original files and reapply your modifications to them?

**Comment by stripeyhorse** [comment]
yes. sorry. i dont know why that changed so much. i only added maybe 5 lines of code..

let me try again on the weekend. ill resync fresh and code it on my linux box.

**Comment by 1technophile** [comment]
no problem !

**Comment by 1technophile** [comment]
@stripeyhorse do you think you could update your pull request ? I will be happy to integrate it once is done

**Comment by stripeyhorse** [comment]
@1technophile yes im actually going to give this a go on the weekend as im going to upgrade to the new codebase and add in the sensors (and i got a few extras to add , light sensor and a temp only high accuracy)

**Comment by 1technophile** [comment]
Hi @stripeyhorse,
since it has been a long time this PR has been done we will have to redo it from the last master code revision. Do you think you would be able to do it?

**Comment by 1technophile** [comment]
I'm closing the PR, if you would like to resubmit it from the last dev branch don't hesitate.
